### PR TITLE
Add base24 support and change repo name to tinted-tmux

### PR DESCRIPTION
**Pull request by JamyGolden**
## Description

Since we're moving towards supporting multiple scheme systems with our repos [tinted-sublime-text](https://github.com/tinted-theming/tinted-sublime-text), [tinted-vscode](https://github.com/tinted-theming/tinted-vscode)  we're going to move towards this with our other template repositories too (ie base16-shell, base16-fzf, and base16-vim soon) since it's easier to maintain and easier for users to have access to the different themes without having to add multiple scheme system repos per template.

It would be easier to read the commits themselves rather than the github `Changes` tab since there are hundreds of updated generated themes, but I've kept them in a single commit.

## PR Changes

- Repository now supports multiple theme systems: base16, base24
- Add base24 template
- Set up [TPM[(https://github.com/tmux-plugins/tpm) plugin to work with multiple theming systems
- Fix bug where TPM doesn't init theme properly
- New tinted TPM variable is named `@tinted-color` (`./tmuxcolors.tmux`), however `@colors-base16` will continue to work for users already using that variable.

## Backwards compatibility

After merging this I will rename the repo from `base16-tmux` to `tinted-tmux` and github will deal with redirecting from https://github.com/tinted-theming/base16-tmux to https://github.com/tinted-theming/tinted-tmux automatically.

Any user who has `base16-tmux` cloned locally should be able to do a `git pull` and carry on without knowing anything has changed aside from getting more themes in `./colors/base24-*`.

### CODE REVIEW

File: templates/base24.mustache [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+# default window title colors
+set-window-option -g window-status-style "fg=#{{base04-hex}},bg=#{{base01-hex}}"
+
+# active window title colors
+set-window-option -g window-status-current-style "fg=#{{base13-hex}},bg=#{{base01-hex}}"
+
+# pane border
+set-option -g pane-border-style "fg=#{{base01-hex}}"
+set-option -g pane-active-border-style "fg=#{{base05-hex}}"
```
Comment (FredHappyface) [inline templates/base24.mustache]: curious on the reason for updating base04 to base05 here? :)

File: templates/base24.mustache [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+# message text
+set-option -g message-style "fg=#{{base10-hex}},bg=#{{base12-hex}}"
```
Comment (FredHappyface) [inline templates/base24.mustache]: Also curious on this change as its the following in base16

```
set-option -g message-style "fg=#{{base06-hex}},bg=#{{base02-hex}}"
```

**Review by FredHappyface — APPROVED** [review]
couple of questions (more for my own learning than anything) though I'm very happy with the changes here - thank you :)

File: templates/base24.mustache (hunk 1 above)
Comment (JamyGolden) [reply templates/base24.mustache]: I'm not sure actually, I think it was a mistake, I'll revert.

File: templates/base24.mustache (hunk 2 above)
Comment (JamyGolden) [reply templates/base24.mustache]: I think I was debugging and left it in, I'll revert.

**Comment by JamyGolden** [comment]
@FredHappyface thanks for looking into it so thoroughly! I was doing some styling to try to give something different to base24 themes, but in doing so I realised there really isn't that much to style in tmux haha. Added responses to your other comments.

File: templates/base24.mustache (hunk 2 above)
Comment (FredHappyface) [reply templates/base24.mustache]: Ah ok cool. I didn't know if it was a case of taking an opinionated approach to messages? A while ago I created a few opinionated 'flavours' for a Qt theme

**Pull request merged**